server/start.go

```diff
@@ -296,6 +296,16 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 			WithHomeDir(home).
 			WithChainID(genDoc.ChainID)
 
+		if config.GRPC.Enable {
+			// If grpc is enabled, configure grpc client for grpc gateway.
+			grpcClient, err := grpc.Dial(config.GRPC.Address, grpc.WithInsecure())
```

Comment:
This is how gRPC clients should be instantiated: https://github.com/cosmos/cosmos-sdk/blob/master/server/grpc/server_test.go#L65

WithInsecure does not matter since the node is open and hence the connection is insecure. ForceCodec matters otherwise it will panic on certain cases.